.kube-linter.yaml

```diff
@@ -0,0 +1,9 @@
+checks:
+  exclude:
+  - "no-read-only-root-fs"
+  - "run-as-non-root"
+  - "unset-cpu-requirements"
+  - "unset-memory-requirements"
+  - "no-anti-affinity"
+  - "latest-tag"
```

Comment:
I'm not sure I like that but that's how the app is already 👍🏻

Reply:
Yes I asked myself the same question.
Let's begin with this config and we'll tune later...